tempCodeRunnerFile.js

```diff
@@ -0,0 +1,14 @@
+
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
+console.log(getRandomPrice());
```

Comment:
@carl12 @SimonC989 Not sure what this is. Probably shouldn't be merged into master.